packages/frontend/src/components/organisms/admin/CreateUserDialog.tsx

```
@@ -0,0 +1,106 @@
+            <form
+              onSubmit={(e) => {
+                e.preventDefault();
+                e.stopPropagation();
+                form.handleSubmit();
+              }}
+            >
+              <Dialog.Body>
+                <VStack>
+                  <form.Field
+                    name="username"
+                    children={(field) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)